app/src/ui/organism/product-category/ProductCategory.js

```diff
@@ -87,7 +80,7 @@ const ProductCategory = () => {
         <div className="es-pt-6">
           <div className="row gy-4">
```

Comment:
if you cut this classes "row gy-4" and put them to parent in line 80 then you can remove this this div (one less nesting)

Reply:
ok. deleted one div and moved className to div above